client/my-sites/theme/controller.jsx

```diff
@@ -2,43 +2,25 @@
  * External Dependencies
  */
 import React from 'react';
-import omit from 'lodash/omit';
 import debugFactory from 'debug';
 import startsWith from 'lodash/startsWith';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal Dependencies
  */
 import ThemeSheetComponent from './main';
 import ThemeDetailsComponent from 'components/data/theme-details';
 import { getCurrentUser } from 'state/current-user/selectors';
-import { getThemeDetails } from 'state/themes/theme-details/selectors';
 import {
 	receiveThemeDetails,
 	receiveThemeDetailsFailure,
 	setBackPath
 } from 'state/themes/actions';
 import wpcom from 'lib/wp';
 import config from 'config';
-import { decodeEntities } from 'lib/formatting';
 
 const debug = debugFactory( 'calypso:themes' );
-let themeDetailsCache = new Map();
-
-export function makeElement( ThemesComponent, Head, store, props ) {
```

Comment:
Ahh yes :( . Function added to `themes/controller.jsx` - it is just a helper function for that file now. Is there a reason that it should not be used this way(or in any way)? - @ockham wants to get rid of this function but i think it was horrible only in context of `Theme Sheet`, or there is more complicated reason?